packages/flutter/lib/src/animation/tween_sequence.dart

```diff
@@ -88,14 +88,49 @@ class TweenSequence<T> extends Animatable<T> {
         return _evaluateAt(t, index);
     }
     // Should be unreachable.
-    assert(false, 'TweenSequence.evaluate() could not find a interval for $t');
+    assert(false, 'TweenSequence.evaluate() could not find an interval for $t');
     return null;
   }
 
   @override
   String toString() => 'TweenSequence(${_items.length} items)';
 }
 
+/// Enables creating a flipped [Animation] whose value is defined by a sequence
+/// of [Tween]s.
+///
+/// This creates a [TweenSequence] that evaluates to a result that flips the
+/// tween both horizontally and vertically.
+///
+/// This tween sequence assumes that the evaluated result has to be a double
+/// between 0.0 and 1.0.
+class FlippedTweenSequence extends TweenSequence<double> {
```

Comment:
Maybe we should also provide (separate PR) a FlippedTween, like ReverseTween:
```dart
class FlipTween<T> extends Tween<T> {
  FlipTween(this.parent)
    : assert(parent != null),
      super(begin: parent.end, end: parent.begin);

  final Tween<T> parent;

  @override
  T lerp(double t) => 1.0 - parent.lerp(1.0 - t);
}
```

Reply:
Created an issue to track this: https://github.com/flutter/flutter/issues/43011